lib/LabeledTextItem/LabeledTextItem.less

```diff
@@ -1,18 +1,14 @@
 /* Labeled Text Item */
 .moon-labeledtextitem {
 	min-width: 336px;
-	height: 192px;
 	overflow: hidden;
-	margin: 0px;
 }
 
 /* Label */
 .moon-labeledtextitem .label {
 	.moon-sub-header-text;
 	color: @moon-label-text-color;
-	border-top: 3px solid @moon-label-border-color;
-	margin: 0px 0px 3px 0px;
-	padding: 6px 0px 0px 0px;
+	margin: 0px 0px 6px 0px;
```

Comment:
This line can just specify: `margin-bottom: 6px`. We don't need to explicitly assign 0px to top, right, or left. Thanks!